internal/translator/utils/rules.go

```diff
@@ -48,12 +52,32 @@ func FilterRules(rules []data.Rule, words []linguakit.Word) data.Rule {
 }
 
 // FilterRuleByType returns the total coincidences  between rule details type and linguakit words type
-func FilterRuleByType(rule data.Rule, words []linguakit.Word) int {
+func FilterRuleByType(rule data.Rule, words []linguakit.Word) (int, int) {
 	var coincidence = 0
+	var properties = 0
 
 	for i := 0; i < len(rule.Details); i++ {
 		if rule.Details[i].Type == words[i].Type {
 			coincidence++
+			if rule.Details[i].Properties != nil {
+				properties += FilterRuleByProperties(rule.Details[i].Properties, words[i].Properties)
+			}
+		}
+	}
+
+	return coincidence, properties
+}
+
+// FilterRuleByProperties returns the total coincidences  between rule details properties and linguakit words properties
+func FilterRuleByProperties(ruleProperties map[string]string, wordProperties map[string]string) int {
+	var coincidence = 0
+
+	for rProperty, rValue := range ruleProperties {
+		for wProperty, wValue := range wordProperties {
+			if rProperty == wProperty && rValue == wValue {
```

Comment:
For clean code it's better avoid prefixes with `w`, `r`

It's better to use the whole words
e.g
`wordProperty` and `ruleProperty`.